ut/s3_delete_object_action_test.cc

```diff
@@ -147,6 +147,7 @@ class S3DeleteObjectActionTest : public testing::Test {
   int call_count_one;
   int layout_id;
   std::string bucket_name, object_name;
+  bool is_object_delete_marker = false;
```

Comment:
Are you using this variable?

Reply:
is_object_delete_marker 
during cleanup if object_metadata is delete marker 
then we need to mark operation done

but during cleanup metadata is already deleted 
so variable used to to check if it is delete marker